light-zk.js/src/test-utils/initLookUpTable.ts

```diff
@@ -137,21 +139,21 @@ export async function initLookUpTableTest(
       REGISTERED_VERIFIER_TWO_PDA,
       MINT,
     ];
-
+    extraAccounts = extraAccounts.concat(additonalAccounts);
     if (extraAccounts) {
       for (var i in extraAccounts) {
         addressesToAdd.push(extraAccounts[i]);
       }
     }
 
-    const extendInstruction = AddressLookupTableProgram.extendLookupTable({
-      lookupTable: lookupTableAddress,
-      authority: payerPubkey,
-      payer: payerPubkey,
-      addresses: addressesToAdd,
-    });
+    // const extendInstruction = AddressLookupTableProgram.extendLookupTable({
```

Comment:
Any reason it's commented, but not removed?

Reply:
Yeah I discovered we have two functions which create lookup tables. I want remove this one in another pr.